js/members.js

```diff
@@ -67,9 +76,52 @@ $("#batchButtons button").on("click", function() {
     $(this).addClass("active-batch");
 });
 
+// funtion to highlight selected tab on "allMembersButtons" Bar
+$("#allMembersButtons button").on("click", function() {
+    var buttons = $('#allMembersButtons').children();
+    for (button of buttons) {
+        $(button).removeClass("active-batch");
+    }
+    $(this).addClass("active-batch");
+   
+    // if "presentMembers" tab selected, make "all" as selected Tab on "batchButtons"
+    if(this == buttons[0])
+    {
+        let allbuttons = $('#batchButtons').children();
+        for (allbutton of allbuttons) {
+            $(allbutton).removeClass("active-batch");
+        }
+        $(allbuttons[0]).addClass("active-batch");
+    }
+
+    //if "Alumni" tab selected, make "all" as selected Tab on "alumniButtons"
+    else if(this == buttons[1])
+    {
+        let allbuttons = $('#alumniButtons').children();
+        for (allbutton of allbuttons) {
+            $(allbutton).removeClass("active-batch");
+        }
+        $(allbuttons[0]).addClass("active-batch");
+    }
+});
+
+// function to highlight selected on alumniButtons Bar
+$("#alumniButtons button").on("click", function() {
+   var buttons = $("#alumniButtons").children();
+   for (button of buttons) {
+       $(button).removeClass("active-batch");
+   }
+   $(this).addClass("active-batch");
+});
+
+
 // Event Listeners
-function clickAll() {
+function clickAllPresentMembers() {
     $('#allMembersContainer').html(members['fourthYears']+members['thirdYears']+members['secondYears']);
+    //hide alumniButton Bar
+    $("#alumniButtons button").css("display", "none");
```

Comment:
Why don't you just set ```display: none``` on the container div itself instead of on every single child button?

Reply:
done